src/calculateRentalCost.js

```diff
@@ -4,7 +4,26 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
```

Comment:
Prefer defining `pricePerDay` as a constant outside of the function to avoid re-declaration on every function call and to follow the checklist item 2 for non-hardcoded values.